resourcematch-api/main.py

```diff
@@ -96,18 +107,19 @@ async def generate_search_queries(openai_client, model, text):
     return result['search_queries']
 
 
-async def process_search_queries(search_queries, books):
+async def process_search_queries(search_queries, subject):
```

Comment:
Should we rename this argument since it will be a list of books slugs? Or, perhaps we want to just pass the subject and do something like `books = BOOK_SLUGS_BY_SUBJECT[subject]` in this function?

Reply:
You're right - it should be `books` because we are passing `BOOK_SLUGS_BY_SUBJECT[subject]` in the handler itself. I'll change it back.